airflow/utils/db.py

```diff
@@ -200,6 +200,10 @@ def initdb():
         Connection(
             conn_id='spark_default', conn_type='spark',
             host='yarn', extra='{"queue": "root.default"}'))
+    merge_conn(
+        Connection(
+            conn_id='livy_default', conn_type='livy',
+            host='livy', port=8998))
```

Comment:
make port configurable. none of our Livy instances run on the standard port

Reply:
This does make it configurable - this is just the default connection in Airflow.